cdc/tests/integration_tests/flow_control/run.sh

```diff
@@ -39,13 +42,15 @@ EOF
 	esac
 
 	tikv-cdc cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI"
-	rawkv_op $UP_PD put 100000 # About 30M
+	# Wait until cdc pulls the data from tikva nd store it in soter
+	sleep 90
 
 	rss1=$(ps -aux | grep 'tikv-cdc' | head -n1 | awk '{print $6}')
 	if [[ $rss1 == "" ]]; then
+		echo "Failed to get rrs by ps"
 		exit 1
 	fi
-	expected=1048576 # 1M
+	expected=524288 # 1G
```

Comment:
```suggestion
	expected=524288 # 512M
```

Is it reasonable that the memory usage increased by `512MB` even if we specify memory quota as `10MB` ? If yes, suggest to explain the reason here.

Reply:
For log, the memory used about `150M - 200M` with flow-control, I modified the `expected=200M`. I think processor also needs part of memory for structs and gorountine. Do you think this is reasonable？

Reply:
OK, that makes sense.